src/System.Net.Sockets/src/System/Net/Sockets/NetworkStream.cs

```diff
@@ -614,6 +614,9 @@ internal bool Connected
         // Returns:
         // 
         //     An IASyncResult, representing the read.
+#if !netcore50
+        override
+#endif
```

Comment:
Dropping netcore50 support in the build sounds fine, but I'd want to make sure it won't interfere with work elsewhere (@CIPop and @davidsh are working on UWP networking stuff now).  Perhaps we should keep this as-is for now, and file an issue to change it later?